migration/migrator/main.py

```diff
@@ -470,3 +473,66 @@ def dump(args):
         semester = f"{'s' if today.month < 7 else 'f'}{str(today.year)[-2:]}"
         dump_database(f'submitty_{semester}_sample', data_dir / 'course_tables.sql')
         print('DONE')
+
+
+def load_triggers(args, silent=False):
+    for environment in args.environments:
+        if environment != 'master' and environment != 'course':
+            continue
+
+        trigger_dir = get_triggers_path() / environment
+        if not trigger_dir.is_dir():
+            if silent:
+                continue
+            raise SystemExit('Error: Could not find triggers directory for ' + environment)
```

Comment:
This error condition should only happen if someone were to delete the `course` or `master` directory within the repo. I don't think this is an error case that would happen in production unless someone was purposely trying to break their install and if it happens in development, fine to have a noisy error raised from when do `trigger_dir.iterdir()`.

As such, I'd remove this as it adds unnecessary complexity.